### PR TITLE
Add custom media queries

### PR DESCRIPTION
**Pull request by devnacho**
Hey @rtfeldman 

This PR is for making custom media queries like:

```elm
    mediaQuery "screen and (max-width: 480px)"
        [ body [ margin (em 3) ] ]
```

Let me know if everything looks good.

Thanks for the help!!

### CODE REVIEW

File: src/Css.elm [hunk 1]
```diff
@@ -4220,6 +4225,11 @@ color c =
 
 
 {-| -}
+mediaQuery : String -> List Snippet -> Snippet
+mediaQuery queryString snippets =
+  media [ customMediaQuery queryString ] snippets
```
Comment (rtfeldman) [inline src/Css.elm]: Can we change this to `media [ MediaQuery queryString ] snippets`? I think we can do without the helper function here! 😄

**Comment by rtfeldman** [comment]
Great! I added one very minor comment, but otherwise looks 👍!

**Comment by devnacho** [comment]
@rtfeldman makes sense! without the helper function is more straightforward.

I made the changes!

**Comment by rtfeldman** [comment]
Woohoo! Thanks @devnacho!

**Comment by rtfeldman** [comment]
Published as 2.1.0

**Comment by devnacho** [comment]
@rtfeldman awesome!!

**Pull request merged**